### PR TITLE
Add deployment information to mkdocs, and reference eoapi-template.

### LINKED ISSUE #113: Include deployment options in docs as well.
We should probably include the deployment option additions from https://github.com/developmentseed/eoAPI/pull/111 in the docs pages as well as just the README.

I'd also suggest adding the https://github.com/developmentseed/eoapi-template in both as well.

### PR DESCRIPTION
**Pull request by zacharyDez**
Closes #113 

Based on @sharkinsspatial recommendations:
- Added reference to eoAPI-template
- Updated mkdocs/intro to match README

### CODE REVIEW

File: docs/src/intro.md [hunk 1]
```diff
@@ -1,68 +1,46 @@
----
-hide:
-  - navigation
----
-
 
 <p align="center">
-  <img width="700" src="../img/eoAPI.png"/>
+  <img width="700" src="../../docs/logos/eoAPI.png"/>
```
Comment (vincentsarago) [inline docs/src/intro.md:3]: Why is this needed ?

File: docs/src/deployment.md [hunk 2]
```diff
@@ -3,6 +3,10 @@ hide:
   - navigation
 ---
 
+The default runtimes are available through separate repositories ([eoapi-cdk](https://github.com/developmentseed/eoapi-cdk) and [k8s-eoapi](https://github.com/developmentseed/k8s-eoapi)). 
+
+A demonstration application is accessible with the repository [eoapi-template](https://github.com/developmentseed/eoapi-template).
+
```
Comment (vincentsarago) [inline docs/src/deployment.md:9]: I think we should really explain clearly what are the option for deploying an eoAPI like system. 

right now this page `deployment.md` is a mix of two things:
- links to external repo (eoapi-cdk and k8s-eoapi)
- a step-by-step `how to` to deploy eoapi custom demo application 

IMO this is really confusing. 

### Possible solution
1. create an `eoapi-docs` repo to host all the documentation about eoapi
2. use the current `eaopi` repo as an example of how eoapi can work

cc @sharkinsspatial @zacharyDez

File: docs/src/intro.md [hunk 3]
```diff
@@ -78,5 +56,4 @@ export DATABASE_URL=postgresql://username:password@0.0.0.0:5439/postgis  # Conne
 .venv/bin/uvicorn eoapi.{SERVICE}.app:app --port 8000 --reload
 ```
 
-Note: services might have incompatible dependencies which you can resolve by using virtual environement per service
-
+Note: services might have incompatible dependencies which you can resolve by using a virtual environment for each service
```
Comment (vincentsarago) [inline docs/src/intro.md:59]: I'm confused by the overall change in this page. When I `refactored` it I tried to split the documentation from the eoapi demo application (hosted in eoapi repo) to just give an overview of what was the eoapi concept and why people should use it

File: docs/src/deployment.md (hunk 2 above)
Comment (ranchodeluxe) [reply docs/src/deployment.md:9]: I just renamed the repo to `eoapi-k8s` so that will have to change too

File: docs/src/intro.md (hunk 3 above)
Comment (zacharyDez) [reply docs/src/intro.md:59]: @vincentsarago should we just revert the intro to your previous version but keep the changes in `deployment.md`?

File: docs/src/deployment.md (hunk 2 above)
Comment (zacharyDez) [reply docs/src/deployment.md:9]: @vincentsarago We can also just remove the external references if you prefer, but I thought it would be helpful for someone interested in deployment to have this information readily available.

File: docs/src/intro.md (hunk 3 above)
Comment (zacharyDez) [reply docs/src/intro.md]: done!

File: docs/src/deployment.md (hunk 2 above)
Comment (sharkinsspatial) [reply docs/src/deployment.md:9]: @vincentsarago I agree this is confusing.  Ideally I think the eoAPI repo would have 3 deployment sections

1. How to deploy the eoAPI demonstration runtimes locally via docker.
2. A description of `eoapi-cdk` , `eoapi-cdk` and `eoapi-template` and pointers to their docs for deploying with the default runtimes they contain.
3. How to deploy the eoAPI demonstration runtimes using a local CDK app (mostly copy and pasted from `eoapi-template`)  that uses the `eoapi-cdk` constructs but with runtime overrides.

I think this is what @zacharyDez was shooting for with his PR but based on your feedback maybe we need a bit more clarification.

Comment (vincentsarago) [reply docs/src/deployment.md:9]: @sharkinsspatial yep that's it on point!

Comment (zacharyDez) [reply docs/src/deployment.md:9]: Got it! Will go for another iteration.

**Review by vincentsarago — APPROVED** [review]
